### PR TITLE
Adds Discourse link to experiment details page

### LINKED ISSUE #1019: Add Discourse Link to each experiment details page
You know, in the details section

### PR DESCRIPTION
**Pull request by kaganjd**
- Fixes #1019
- updates frontend view, template, and test for experiment page
- updates experiment model, serializer, and test
- adds migration for model update
- adds discourse key to app.l20n
- updates `fixtures/demo_data.json` with `discourse_url`

### CODE REVIEW

File: testpilot/frontend/static-src/app/templates/experiment-page.js [hunk 1]
```diff
@@ -52,6 +52,11 @@ export default `
                       <td data-l10n-id="bugReports">Bug Reports</td>
                       <td><a data-hook="bug-report-url"></a></td>
                     </tr>
+
+                    <tr>
+                      <td data-l10n-id="discourse">Discourse</td>
```
Comment (meandavejustice) [inline testpilot/frontend/static-src/app/templates/experiment-page.js:57]: cc/ @johngruen Are you cool with this copy? Would you rather have something like "forum" or "discuss" here?

**Comment by meandavejustice** [comment]
@kaganjd This is great! I'm just gunna wait for john to respond on the comment above regarding copy, then I'm good to merge.

**Comment by meandavejustice** [comment]
Just gunna merge this, we can make copy updates later

**Comment by jaredhirsch** [comment]
:tada: Nice! I'm really excited about this change ^_^

**Pull request merged**